src/Game.ts

```diff
@@ -224,6 +224,27 @@ export abstract class Game {
     return IsInputDisabled(2) ? InputMode.MouseAndKeyboard : InputMode.GamePad;
   }
 
+  /**
+   * Gets the player's current radio station.
+   *
+   * @returns A radio station.
+   */
+  public static get RadioStation(): RadioStation {
+    const stationName: string = GetPlayerRadioStationName();
+    const keys = Object.keys(RadioStation).filter(x => RadioStation[x] === stationName);
```

Comment:
Unfortunately, TypeScript doesn't support reverse mapping of string enums, so you have to use a bit of a hack here.